README.md

```diff
@@ -1,2 +1,28 @@
 # LinQuadOptInterface.jl
-Intermediate wrapper for MOI of CPLEX, Gurobi and Xpress
+
+LinQuadOptInterface (LQOI) is designed to make it easier for low-level wrapper designed to bridge low-level Integer Linear and Quadratic solvers to implement the MOI interface. It provides access to many MathOptInterface functionalities mainly related to problem modifications: 
+
+1. add constraints/variables 1-by-1 and in batches 
+2. remove constraints/variables
+3. change constraint coefficients 
+4. change constraint type
+5. change constraint rhs
+6. change variable bounds and type
+
+## LinQuadOptInterface Instance
+
+In the MOI `Instance` made available by LinQuadOptInterface.jl all these modification are caried out by the low-level solver own functionalities and hence it differs from MathOptInterfaceUtilities' `Instance`. The latter will keep all problem data in the julia level and typically push to a low-level solver the complete problem at once.
```

Comment:
`Instance` does not directly connect to solvers. The comparison here is with solver wrappers that *use* `Instance` internally.

Reply:
What about:
The MOI `Instance` made available by LinQuadOptInterface.jl is used as a standardized interface to a solver which has a low-level wrapper that allows most of (or all) these modification to be made in the solver's internal structures. The data kept in the julia level is minimal.

Reply:
Solver instance, not instance, right? It differs from other wrappers that store a standalone instance in Julia.

Reply:
Now I understand what you mean, I will re-do it more clearly